### ISSUE TITLE
Flow reports zero errors on a file with errors in most basic setup

### ISSUE THREAD

**Issue by AndrewRayCode** [facebook/flow#3898]
Install:

    $ yarn add flow flow-bin flow-cli
        ├─ flow@0.2.3
        ├─ flow-bin@0.45.0
        └─ flow-cli@0.0.0-pre

A file full of errors:

    $ cat test.js

        const a: string = 'hi';

        const c = a.map(item => {
            return item.toString();
        });

        const b = 2;
        b.map(item => {
            return item.toString();
        });

        const b = 2 + 'a' - b();

Most basic flow configuration? This may be the problem as the documentation for `.flowconfig` is hard to follow and unclear about how to correctly set up a project

    $ cat .flowconfig

        [include]
        test.js
        ./test.js
        .

Add flow to path for quick testing

    $ export PATH="./node_modules/.bin:$PATH"

Run flow?

    $ flow
        Launching Flow server for flow-test
        Spawned flow server (pid=13154)
        Logs will go to /private/tmp/flow/zSUserszSyzSflow-test.log
        No errors!

Uhhhh?

    $ flow
        No errors!
        
Uhhhh?

    $ cat /private/tmp/flow/zSUserszSyzSflow-test.log

        [2017-05-07 14:31:54] Initializing Server (This might take some time)
        [2017-05-07 14:31:54] executable=flow-test/node_modules/flow-bin/flow-osx-v0.45.0/flow
        [2017-05-07 14:31:54] version=0.45.0
        [2017-05-07 14:31:54] Parsing
        [2017-05-07 14:31:55] Building package heap
        [2017-05-07 14:31:55] Loading libraries
        [2017-05-07 14:31:56] Running local inference
        [2017-05-07 14:31:56] Calculating dependencies
        [2017-05-07 14:31:56] Merging
        [2017-05-07 14:31:56] Done
        [2017-05-07 14:31:56] Server is READY
        [2017-05-07 14:31:56] Took 1.494020 seconds to initialize.
        [2017-05-07 14:31:56] Status: OK
        [2017-05-07 14:36:33] Status: OK
        [2017-05-07 14:37:11] Status: OK

Flow running? Yup, it's trying it's hardest to [crash my computer again](https://github.com/facebook/flow/issues/2152):

    $ ps aux | grep flow
        13156   0.0  0.0 29730704   5852   ??  S     2:31PM   0:00.02 flow-test/node_modules/flow-bin/flow-osx-v0.45.0/flow
        13155   0.0  0.0 29729680   5832   ??  S     2:31PM   0:00.01 flow-test/node_modules/flow-bin/flow-osx-v0.45.0/flow
        13154   0.0  0.4 29787812  65724   ??  Ss    2:31PM   0:01.00 flow-test/node_modules/flow-bin/flow-osx-v0.45.0/flow
        13465   0.0  0.0  2432804    776 s000  S+    2:40PM   0:00.00 grep flow
        13212   0.0  0.2  2683144  41348   ??  Ss    2:32PM   0:11.94 /Applications/MacVim.app/Contents/MacOS/Vim -f -g /private/tmp/flow/zSUserszSyzSflow-test.log
        13159   0.0  0.0  2483364   8128   ??  S     2:31PM   0:00.07 flow-test/node_modules/flow-bin/flow-osx-v0.45.0/flow
        13158   0.0  0.0 29730704   5840   ??  S     2:31PM   0:00.01 flow-test/node_modules/flow-bin/flow-osx-v0.45.0/flow
        13157   0.0  0.0 29729680   5840   ??  S     2:31PM   0:00.02 flow-test/node_modules/flow-bin/flow-osx-v0.45.0/flow

Since Flow locks up my computer on an existing project, I wanted to try a bare-bones installation. But flow doesn't seem to want to run against my source code.

For a sanity check, let's try a default setup. Surely default flow will analyze Javascript files in the current directory:

    $ rm .flowconfig
    $ flow init
    $ flow
        Launching Flow server for flow-test
        Spawned flow server (pid=13518)
        Logs will go to /private/tmp/flow/zSUserszSyzSflow-test.log
        No errors!

Nope?

**Comment by madbence** [comment]
you have to annotate your files with `// @flow`

**Comment by AndrewRayCode** [comment]
Whoops! thanks
